cmd/kubeadm/app/preflight/checks.go

```diff
@@ -627,6 +674,9 @@ func RunInitMasterChecks(cfg *kubeadmapi.MasterConfiguration) error {
 			ControllerManagerExtraArgs: cfg.ControllerManagerExtraArgs,
 			SchedulerExtraArgs:         cfg.SchedulerExtraArgs,
 		},
+		HTTPProxyCheck{Proto: "https", Host: cfg.API.AdvertiseAddress, Port: int(cfg.API.BindPort)},
```

Comment:
is this still necessary?

Reply:
Yes, that is another check. just changed order, so all proxy related warnings will be in one place and not mixed with warnings/errors of something else.